free-1.md

```diff
@@ -57,12 +107,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I don't think there will be a lot of language decisions -- the input I need only requires 4 basic pieces of information from the user, probably in the form of some combination of text boxes / drop-down menus / check boxes.
 
```

Comment:
I agree, the language component seems much smaller than the "systems" aspect.  Perhaps you could extend the language by allowing the user to indicate the time signature, length of the note, and rests to create a very simple score.  However, this would probably end up adding even more to the systems aspect.  Also, you might decide that these extensions go outside of your domain, and would be the job of a more extensive application like MuseScore.